recipe/conda_build_config.yaml

```diff
@@ -6,6 +6,7 @@ python_abi_tag:
  - cp311
  - pypy38_pp73
  - pypy39_pp73
+ - graalpy_38_native
```

Comment:
I will add a patch to the graalpy build to use `graalpy223_38_native` as abi tag. We keep breaking the ABI even across minor releases, and this is just a mistake on our side that we aren't encoding that. I'll fix it upstream and add the backported patch to the feedstock PR.

Reply:
Please consider the tag name carefully: this is the project's chance to determine what it should be. Do you want to save a few letters in the shared object and wheel names by using `gp` instead of `graalpy` or otherwise change the designation? Are you sure you want a `223` in the tag: how often will this change and necessitate a rebuild of all the conda binary packages?

Reply:
Patch added here: https://github.com/conda-forge/staged-recipes/pull/20882/commits/0209399df6ebf1f8be4300eb6594dd71d0e091da

Now our EXT_SUFFIX is e.g. `'.graalpy223-38-native-x86_64-linux.so'`. This change will go out to github master of graalpy today as well (and be there for the next version, which will say `'.graalpy230-310-native-x86_64-linux.so'`)

Reply:
> Please consider the tag name carefully: this is the project's chance to determine what it should be. Do you want to save a few letters in the shared object and wheel names by using `gp` instead of `graalpy` or otherwise change the designation?

I don't see the benefit of saving 5 bytes going `graalpy` -> `gp`.

> Are you sure you want a `223` in the tag:

The version name I would like in the tag, yes. We do break ABI pretty much every release, but since we control the tag on our end, if we end up not breaking it anymore, we can always drop that number or keep it stable as the latest ABI version release.

> how often will this change and necessitate a rebuild of all the conda binary packages?

I would expect we update Conda only once a year with the latest release of GraalPy. That wouldn't be different from CPython, right, at least for packages that don't build against only the limited ABI, right?